server/controllers/checklist.js

```diff
@@ -0,0 +1,129 @@
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const { v4: uuidv4 } = require("uuid");
+
+exports.createChecklist = asyncHandler(async (req, res, next) => {
+  const { checklistItem, cardId, columnId, boardId, userId } = req.body;
+  if (!userId || !boardId || !columnId || !cardId || !checklistItem) {
+    res.status(400);
```

Comment:
Hey, let's actually keep this here for when we implement our plugins. We can re-use this 😄